release/platform/platform.go

```diff
@@ -367,4 +367,12 @@ var platforms = []Platform{
 		Repos:     []string{RepoOrg, RepoEnterprise},
 		BuildTags: []string{"ssl", "sasl", "gssapi", "failpoints"},
 	},
+	{
+		Name:      "rhel-82",
+		Arch:      "arm64",
```

Comment:
I think arch should be "aarch64". In RHEL world its called "aarch64", but in Debian\Ubuntu its indeed "arm64"

Reply:
```
root@removed:~ $ rpm --eval '%{_arch}'
aarch64
```